Configuration/TypoScript/TrackingObjects/MicrosoftStream.typoscript

```diff
@@ -0,0 +1,50 @@
+plugin.tx_cookieman.settings.trackingObjects {
+    MicrosoftStream {
+        # inline scripts would break Content-Security-Policy: script-src
+        # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src
+        show {
+            # https://www.microsoft.com/de-de/trust-center/privacy/gdpr-overview
+            MicrosoftStream {
+                duration =
+                durationUnit = none
+                type = connection
+                provider = MicrosoftStream
+            }
+            
+            Signature_Api {
+                duration =
+                durationUnit = session
+                type = cookie_http
```

Comment:
Are these really just HTTP cookies? If it's set via JavaScript, it would be a cookie_http+html. That would allow us to delete it when consent is revoked later.
Please confirm also for MicrosoftForms.